docs/framework/resources/packaging-and-deploying-resources-in-desktop-apps.md

```diff
@@ -122,10 +123,39 @@ Applications rely on the .NET Framework Resource Manager, represented by the <xr
   
 -   If the probe for a particular resource assembly fails, the runtime does not raise the <xref:System.AppDomain.AssemblyResolve?displayProperty=nameWithType> event.  
   
+
+### .NET Core resource fallback process
+ The .NET Core resource fallback process involves the following steps:
+
+1.  The runtime attempts to load a satellite assembly for the requested culture, eventually calling <xref:System.Runtime.Loader.AssemblyLoadContext.Load(System.Reflection.AssemblyName)?displayProperty=nameWithType> on the AssemblyLoadContext containing the the currently executing assembly.  The <xref:System.Runtime.Loader.AssemblyLoadContext.Default?displayProperty=nameWithType>:
```

Comment:
> the the currently executing assembly

Remove the duplicate "the"

Reply:
Should the AssemblyLoadContext be marked as code? - that is `AssemblyLoadContext`.

Reply:
Also I don't really follow the logic of this paragraph: it is basically
> The runtime eventually calls ALC.Load on the right context. The ALC.Default does this:

There doesn't seem to be any logic connection between ALC.Load and the immediately following description of ALC.Default.

I think it should say something like: If the AssemblyLoadContext is the default one or it falls back to the default, then the runtime will: